WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -827,10 +827,10 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
             // It assumes this is being called when the editor has finished loading
             // If you need to refactor this, please ensure that the startup_time_ms property
             // is still reflecting the actual startup time of the editor
-            editorSession.start(unsupportedBlocks: unsupportedBlockNames)
+            editorSession.start(unsupportedBlocks: unsupportedBlockNames, canViewEditorOnboarding: self.canViewEditorOnboarding())
```

Comment:
```suggestion
            editorSession.start(unsupportedBlocks: unsupportedBlockNames, canViewEditorOnboarding: canViewEditorOnboarding())
```

Reply:
While `self` can be used here, it's not needed. It's only normally needed inside closures.

Reply:
Ah, OK. Thanks for the tip. I'll update all the relevant calls to remove `self`.